Create ui-c8y-1019-22-3-fixed-breadcrumbs-for-global-and-inventory-roles-tabs-under-accounts-roles.md (#1992)
SoftwareAG/c8y-docs: content/change-logs/platform-services/ui-c8y-1019-22-3-fixed-breadcrumbs-for-global-and-inventory-roles-tabs-under-accounts-roles.md

---
date: ""

> title: Fixed breadcrumbs for Global roles and Inventory roles tabs

Reviewer: ```suggestion
title: Fixed location information for Global roles and Inventory roles tabs
```

> title: Fixed location information for Global roles and Inventory roles tabs under Accounts > Roles menu

Reviewer: ```suggestion
title: Fixed breadcrumbs for Global roles and Inventory roles tabs
```

> title: Fixed breadcrumbs for global and inventory roles tabs under Accounts > Roles [GRAFT][release/cd] (#6201)

Author: The Claude AI suggested to use the following title:
```suggestion
title: Fixed breadcrumbs for global and inventory roles tabs under Accounts > Roles
```

product_area: Platform services
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-0UgqXH1Ys
    label: Administration
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-57553
version: 1019.22.3
---

> Fixed breadcrumbs for global and inventory roles tabs under Accounts > Roles [GRAFT][release/cd] (#6201)

Author: The Claude AI suggested to use the following body:
```suggestion
In the Cumulocity IoT user interface, the breadcrumbs for the global and inventory roles tabs under the Accounts > Roles menu were previously incorrect, making navigation confusing for users. This issue has been resolved, and the breadcrumbs now accurately reflect the user's location within the application. This change improves the overall user experience by providing clear and consistent navigation across the roles management screens.
```

> In the Administration application, the location information for the **Global roles** and **Inventory roles** tabs under the **Accounts** > **Roles** menu was incorrect, making navigation confusing for users. This issue has been resolved with the user's location within the application now accurately reflected.

Reviewer: ```suggestion
In the Administration application, the location information for the **Global roles** and **Inventory roles** tabs under **Accounts** > **Roles** was incorrect, making navigation confusing for users. This issue has been resolved with the user's location within the application now accurately reflected.
```

> In the Administration application, the location information for the **Global roles**  and **Inventory roles** tabs under the **Accounts** > **Roles** menu was incorrect, making navigation confusing for users. This issue has been resolved with the user's location within the application now accurately reflected.

Reviewer: ```suggestion
In the Administration application, the location information for the **Global roles** and **Inventory roles** tabs under the **Accounts** > **Roles** menu was incorrect, making navigation confusing for users. This issue has been resolved with the user's location within the application now accurately reflected.
```
